praw/models/reddit/live.py

```diff
@@ -281,6 +281,45 @@ def close(self):
         url = API_PATH['live_close'].format(id=self.thread.id)
         self.thread._reddit.post(url)
 
+    def set(self, title, description='', nsfw=False, resources='',
+            **other_settings):
+        """Set settings for the live thread.
+
+        The following settings are supported:
+
+        :param title: The title of the live thread.
+        :param description: The live thread's description.
+        :param nsfw: Indicate whether this thread is not safe for
+            work.
+        :param resources: Markdown formatted information that is
+            useful for the live thread.
+        :returns: New :class:`.LiveThread` instance (updated settings
```

Comment:
No problem. Sorry for my lack of explanation in this PR.

> If you're up for updating the thread's dict with the new returned values that would be awesome.

Reddit doesn't return updated settings for the `POST .../edit` request. So we may
be able to update `thread.__dict__` with the data used in request, but I'm not sure this is OK.